hdf5.spec

```
@@ -1,29 +1,24 @@
 %global macrosdir %(d=%{_rpmconfigdir}/macros.d; [ -d $d ] || d=%{_sysconfdir}/rpm; echo $d)
 %{!?_fmoddir:%global _fmoddir %{_libdir}/gfortran/modules}
 
-%global daos_major 0
-%global hdf5_commit fa40c6c59af5d9aabd4b478cd02f8a9f7ebf7922
-%define hdf5_sha .gfa40c6c59a
+%global daos_major 1
 
 # Patch version?
 %global snaprel %{nil}
-# HDF5 versions
-%global major 1.12
-%global minor 0
 
 # NOTE:  Try not to release new versions to released versions of Fedora
 # You need to recompile all users of HDF5 for each version change
 Name: hdf5
-Version: %{major}.%{minor}
-Release: 5%{hdf5_sha}%{?dist}
+Version: %{hdf5_major}_%{hdf5_minor}
+Release: %{hdf5_release}%{?relval}%{?dist}
```

The `rc5` that you are putting in the `Release:` here belongs in the `Version:` as it's part of the upstream product version number.

```suggestion
Version: %{hdf5_major}.%{hdf5_minor}~%{hdf5_release}
Release: 1%{?relval}%{?dist}
```